fermenter-mda/pom.xml

```diff
@@ -127,19 +127,11 @@
         <!--
             NB:Should get pulled from maven-plugin-testing-harness, but temporarily being specified directly to resolve
             CVE-2023-37460.  Can be retired when maven-plugin-testing-harness uses plexus-archiver >= 4.8.0.
-
-            commons-compress likely can also go at that point, as the following block is to override the version that
-            comes with plexus-archiver 4.8.0 and has vulnerabilities.
           -->
```

Comment:
I (Informational): brought in at the appropriate version (`1.26.1`) via the updated `plexus-archiver`